code/modules/mob/mob.dm

```diff
@@ -215,6 +216,7 @@
 	return log(2, mob_size / MOB_SIZE_MEDIUM)
 
 /mob/proc/Life()
+	set waitfor = FALSE
```

Comment:
This seems like a very strange addition considering the next line should stop it sleeping.